ts/utils.ts

```diff
@@ -65,6 +65,17 @@ export function randomLeaves(num: number): string[] {
     return leaves;
 }
 
+/**
+ * Generates a random address. Usefully for testing when
+ * you don't need a valid address or contract.
+ *
+ * @returns Randomly generated address.
+ */
+export function randomAddress(): string {
+    const hex = crypto.randomBytes(20).toString("hex");
+    return getAddress(hex);
+}
```

Comment:
Will convert https://github.com/thehubbleproject/hubble-contracts/pull/596/files#diff-f42164895eb06a73792de9958c01fd722add2af3eee446525bc00cbfa61c7757R13-R16 to this once this PR is merged in a future PR.